### PR TITLE
Release notes for 1.12.0

### PR DESCRIPTION
**Pull request by rtwalker**

### CODE REVIEW

**Review by jerryxwu — APPROVED** [review]
nit: ideally fix several places where a bullet point does not end with a period.

File: Changelog.md [hunk 1]
```diff
@@ -25,15 +24,20 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
...
 #### Other notable changes
 
+- RPM packages for CentOS 7 have returned
+- New Homebrew formula available for macOS installation: `brew install timescale/tap/timescaledb-toolkit`
 - [#547](https://github.com/timescale/timescaledb-toolkit/pull/547): Update pgx to 0.5.0. This is necessary for adding Postgres 15 support coming soon.
 - [#571](https://github.com/timescale/timescaledb-toolkit/pull/571): Update CI docker image for pgx 0.5.0.
-- [#599](https://github.com/timescale/timescaledb-toolkit/pull/599): Reduce floating point error when using `stats_agg` in moving aggregate mode
+- [#599](https://github.com/timescale/timescaledb-toolkit/pull/599): Reduce floating point error when using `stats_agg` in moving aggregate mode.
+- [#589](https://github.com/timescale/timescaledb-toolkit/pull/589): Update pgx to 0.5.4.
+- [#594](https://github.com/timescale/timescaledb-toolkit/pull/594): Verify that pgx doesn't generate CREATE OR REPLACE FUNCTION
```
Comment (rtwalker) [inline Changelog.md:33]:
```suggestion
- [#594](https://github.com/timescale/timescaledb-toolkit/pull/594): Verify that pgx doesn't generate CREATE OR REPLACE FUNCTION.
```

File: Changelog.md [hunk 2]
```diff
@@ -25,15 +24,20 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
...
+- [#592](https://github.com/timescale/timescaledb-toolkit/pull/592): Add build script option to install in release mode
```
Comment (rtwalker) [inline Changelog.md:34]:
```suggestion
- [#592](https://github.com/timescale/timescaledb-toolkit/pull/592): Add build script option to install in release mode.
```

File: Changelog.md [hunk 3]
```diff
@@ -25,15 +24,20 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
 
 #### Other notable changes
 
+- RPM packages for CentOS 7 have returned
```
Comment (rtwalker) [inline Changelog.md:27]:
```suggestion
- RPM packages for CentOS 7 have returned.
```

File: Changelog.md [hunk 4]
```diff
@@ -25,15 +24,20 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
...
+- New Homebrew formula available for macOS installation: `brew install timescale/tap/timescaledb-toolkit`
```
Comment (rtwalker) [inline Changelog.md:28]:
```suggestion
- New Homebrew formula available for macOS installation: `brew install timescale/tap/timescaledb-toolkit`.
```

**Review by rtwalker — COMMENTED** [review]
Adding periods

File: Changelog.md [hunk 5]
```diff
@@ -4,19 +4,18 @@
...
+  Both the aggregate form and the function form of `Candlestick` support the following (experimental) accessors (in addition to being re-aggregated via `rollup`):
   `open`, `high`, `low`, `close`, `open_time`, `high_time`, `low_time`, `close_time`, `volume`, `vwap` (Volume Weighted Average Price)
+  *NOTE*: This functionality improves upon and replaces the need for `toolkit_experimental.ohlc` which will be removed in the next release.
 
-  NOTE: This functionality improves upon and replaces the need for `toolkit_experimental.ohlc` which will be removed in the next release.
-
-- New min_n/max_n functions and related min_n_by/max_n_by.  The former is used to get the top N values from a column while the later will also track some additional data, such as another column or even the entire row.  These should give the same results as a normal select with an order by and limit, except they can be composed and combined like other toolkit aggregates.
-
-#### Stabilized features
+- [#596](https://github.com/timescale/timescaledb-toolkit/pull/590): New `min_n`/`max_n` functions and related `min_n_by`/`max_n_by`.
```
Comment (rtwalker) [inline Changelog.md:16]:
```suggestion
- [#590](https://github.com/timescale/timescaledb-toolkit/pull/590): New `min_n`/`max_n` functions and related `min_n_by`/`max_n_by`.
```

**Review by Smittyvb — APPROVED** [review]

**Pull request merged**